dummyfs/dummyfs.c

```diff
@@ -158,8 +200,14 @@ int dummyfs_setattr(void *ctx, oid_t *oid, int type, long long attr, const void
 			break;
 
 		case (atDev):
-			/* TODO: add mouting capabilities */
-			ret = -EINVAL;
+			if (data != NULL && size == sizeof(oid_t) && S_ISDIR(o->mode) && !dummyfs_device(fs, &o->dev)) {
```

Comment:
MISRA, also, if `dummyfs_device` has any side effects, it musn't be in `if` condition
```suggestion
			if ((data != NULL) && (size == sizeof(oid_t)) && S_ISDIR(o->mode) && (dummyfs_device(fs, &o->dev) == 0)) {
```